Modules/Chronometer.md

```diff
@@ -0,0 +1,19 @@
+# Name: Chronometer
+
+## Description:
+Chronometer is a PowerShell script and module profiler that allows you to see line by line execution times of your PowerShell commands.
+
+| Works with | Author(s) | Project URL(s) | Creation Date | Tags |
+|------------|--------|-------------------|---------------|------|
+| PS v5.0+ | Kevin Marquette (<a href="https://kevinmarquette.github.io/blog/?utm_source=PowerShellModulesCentral&utm_medium=markdown" target="_blank">b</a> \| <a href="https://twitter.com/kevinmarquette" target="_blank">t</a>) | [Github](https://github.com/KevinMarquette/Chronometer) | Feb 2017 | Performance, Code Coverage |
```

Comment:
If you don't mind, please write the complete month name and not only the abbreviation.